lib/Jit/LLILCJit.cpp

```diff
@@ -641,3 +641,12 @@ void ObjectLoadListener::getRelocationTypeAndAddend(uint8_t *FixupAddress,
     llvm_unreachable("Unknown reloc type.");
   }
 }
+
+unsigned LLILCJit::getMaxIntrinsicSIMDVectorLength(DWORD CpuCompileFlags) {
+  return GenIR::getPreferIntrinsicSIMDVectorLength();
+}
+
+bool LLILCJitContext::isAltJit() {
+  JitOptions JitOptions(*this);
+  return JitOptions.IsAltJit;
```

Comment:
I'm really confused why you'd need to look at altjit in the reader.  Why would SIMD be gated on whether the JIT was plugged in as the altjit or the main jit.  What would happen if LLILC was linked as the main jit and altjit wasn't set?  Would SIMD work?